frontend/src/app/store/app.actions.ts

```
@@ -50,7 +50,14 @@ export class SetUser {
-    constructor(public chatId: number, public paginationValue: number) {}
+export class Logout {
+    static readonly type = '[Auth] Logout';
+}
+
+export class LogoutSuccess {
+    static readonly type = '[Auth] Logout success';
+}
+
+export class LogoutFail {
+    static readonly type = '[Auth] Logout fail';
 }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)